### ISSUE TITLE
Styled buttons in dialog

### ISSUE THREAD

**Issue by mhbailly** [jquery-ui-bootstrap/jquery-ui-bootstrap#202]
Hello,
I'm trying to put some colors in the dialog buttons. ... But it doesn't work :s
So, is it possible to do that? And How to do it?
An example of what I'm trying to do follows.
Thanks in advance,
mh

buttons: [
            {
                text: "Delete",
                click: function() { 
                },
                class:"ui-button-danger"
            },
            {
                text: "Edit",
                click: function() { 
                },
                class:"ui-button-warning"
            },
            {
                text: "Close",
                click: function() { 
                    $(this).dialog("close");
                }
            }
]

**Comment by dharapvj** [comment]
What you are trying to do is not supported in base jquery ui as well. it allows you to only use [certain jquery CSS classes](http://api.jqueryui.com/button/#theming).

What you can do is replicate css of existing classes into your own classes. as shown below

``` html
<!doctype html>

<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>jQuery UI Dialog - Modal confirmation</title>
  <link rel="stylesheet" href="http://code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css" />
  <script src="http://code.jquery.com/jquery-1.9.1.js"></script>
  <script src="http://code.jquery.com/ui/1.10.3/jquery-ui.js"></script>
  <style>
    .ui-dialog-buttonset .ui-button-spl1{
        background:green;
    }
    .ui-dialog-buttonset .ui-button-spl2{
        background:red;
    }
  </style>

  <script>
  $(function() {
    $( "#dialog-confirm" ).dialog({
      resizable: false,
      height:140,
      modal: true,
      buttons: [
            {
                text: "Delete",
                click: function() { 
                },
                class:"ui-button-spl1"
            },
            {
                text: "Edit",
                click: function() { 
                },
                class:"ui-button-spl2"
            },
            {
                text: "Close",
                click: function() { 
                    $(this).dialog("close");
                }
            }
        ]
    });
  });
  </script>
</head>
<body>

<div id="dialog-confirm" title="Empty the recycle bin?">
  <p><span class="ui-icon ui-icon-alert" style="float: left; margin: 0 7px 20px 0;"></span>These items will be permanently deleted and cannot be recovered. Are you sure?</p>
</div>

<p>Sed vel diam id libero <a href="http://example.com">rutrum convallis</a>. Donec aliquet leo vel magna. Phasellus rhoncus faucibus ante. Etiam bibendum, enim faucibus aliquet rhoncus, arcu felis ultricies neque, sit amet auctor elit eros a lectus.</p>


</body>
</html>
```

Let me know if that's sufficient.

**Comment by mhbailly** [comment]
Hi,
Thank you for your answer.
I tried it and it worked but it didn't work in my project with jquery-ui-bootstrap.
I get the same result (attachment).
If there is no obvious solution, I'll find something else.
mh
![2013-08-18 09 51 21](https://f.cloud.github.com/assets/4570391/982237/95bd4518-07db-11e3-9142-b2914b197f9d.png)

**Comment by gtraxx** [comment]
Hi,
I will correct :)

**Comment by gtraxx** [comment]
I will add support for use:
- ui-button-danger
- ui-button-warning
- ui-button-success
- ui-button-info
- ui-button-inverse

Support Add in 1.0 alpha version

### Example :

![multi-button-dialog-2013-9-6](https://f.cloud.github.com/assets/356674/1094535/1e9fb5b4-16cd-11e3-9049-2b9998d582a7.png)

``` javascript
$('#modal-button').dialog({
    autoOpen: false,
    width: 600,
     buttons: [
        {
            text: "Delete",
            click: function() { 
            },
            class:"ui-button-danger"
        },
        {
            text: "Edit",
            click: function() { 
            },
            class:"ui-button-warning"
        },
        {
            text: "other",
            click: function() { 
            },
            class:"ui-button-inverse"
        },
        {
            text: "Close",
            click: function() { 
                $(this).dialog("close");
            }
        }
    ]
});
```

**Comment by gtraxx** [comment]
Fix and add in 1.0 alpha version :)

**Comment by mhbailly** [comment]
Incredible :) Thanks a lot
